test/e2e/Tests/e2e-tests-readme.md

```diff
@@ -0,0 +1,59 @@
+# End-to-End Test Project
+
+This document provides instructions on how to use the end-to-end (E2E) test project for the Azure Functions Durable Extension.
+
+## Prerequisites
+
+- Visual Studio IDE
+- PowerShell
+- npm/Node
+- .NET SDK
+
+## Running the E2E Tests
+
+### Step 1: Increment the host and worker package versions 
```

Comment:
Is this step necessary if I just want to run the tests without making any changes?

Reply:
Will address with package reference work - this will also involve clearing the local NuGet cache which will make this step redundant